liquibase-core/src/main/java/liquibase/change/core/CreateTableChange.java

```diff
@@ -290,4 +292,12 @@ public String getSerializedObjectNamespace() {
         return STANDARD_CHANGELOG_NAMESPACE;
     }
 
+    public String getTableType() {
+        return tableType;
+    }
+
+    @DatabaseChangeProperty(since = "4.13")
```

Comment:
@MalloD12, hello! Should this value be 4.19 instead of 4.13? Right now we're at 4.18.0, but since this has an XSD change, it cannot be merged prior to a major version rev to 4.19.

Reply:
Hi @XDelphiGrl, I think we can remove this annotation because this method (`setTableType()`) is not being used at the moment. Although, if I didn't get it wrong this is just a read-only property annotation. The definition to use this new property according to the Liquibase version is defined in dbchangelog-latest.xsd

Reply:
Done